invenio_cli/helpers/docker_compose.py

```diff
@@ -95,3 +103,14 @@ def destroy_containers(self):
 
         # Close logging pipe
         logpipe.close()
+
+    def copy(self, src_file, dst_path, container):
```

Comment:
With the class dealing with more than docker-compose and being passed as `docker_helper`, changing the name to `DockerHelper` might be clearer.

Reply:
True :) Fixed.